tutorials/how-to/_posts/2019-08-10-interview.md

```diff
@@ -568,14 +568,14 @@ This is probably the biggest fear around technical interviews, and it's one of t
 - **Ask questions.** If I sound like a broken record, that's because this is one of the best things you can do in an interview. Ask about the input and output. Ask for an example. Repeat what they said in your own words. If you don't know what the question is asking, tell the interviewer! *"So I need to calculate a path between every cat and a mouse. How will that be represented?"*
 - **Talk.** Again, broken record time. Talk about what you're stuck on. *"Okay so it's a 2D array of characters. I'm trying to figure out which cat to start with. Or I could start with a mouse..."* This is infinitely better than awkward silence.
 - **Brainstorm.** Rapid-fire potential data structures and algorithms. If you can't decide which option to go with, talk out loud as you ask yourself questions and cross potential options off the list. *"To represent a path, could I use an array? Maybe, but then I'd need to store a path as a set of points. Could I use a HashMap? Maybe I could create a multimap of each cat's position to the best path to each mouse. Would sorting the positions help at all?"* &nbsp; Think of this as a brainstorming session: it's fine to say an idea and then immediately reject it and move on to something else.
-- **Break the problem down into smaller steps.** I talked about this [above](#break-problems-down-into-smaller-steps), and I think it's the most important skill in programming, and maybe more generally than that. Big questions can be paralyzing. How do you even start? You start by breaking the big question down into a few smaller steps, and then taking those steps on one at a time. Helper functions can make this process more concrete. *"I know I eventually need to calculate the paths for every cat. Before I can do that, I need to calculate the path for a single cat. Let's put that in a helper function..."*
+- [**Break the problem down into smaller steps.**](#break-problems-down-into-smaller-steps), and I think it's the most important skill in programming, and maybe more generally than that. Big questions can be paralyzing. How do you even start? You start by breaking the big question down into a few smaller steps, and then taking those steps on one at a time. Helper functions can make this process more concrete. *"I know I eventually need to calculate the paths for every cat. Before I can do that, I need to calculate the path for a single cat. Let's put that in a helper function..."*
```

Comment:
Can you reword this to either be:

`[Breaking the problem down into smaller steps] is the most important skill...`

or

`[Break the problem down into smaller steps.] I think it's the most important skill...`